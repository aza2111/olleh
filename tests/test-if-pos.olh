/*
 * Positive: if/else
 */

Player edwards = fresh Player();

if (edwards.score > 5):
	print "GoodjobEdwards!";

/*else:
	if (edwards.score > 2):
		print "Not too shabby";

	else:
		print "Meh, I've seen better";
	end;*/
end;
