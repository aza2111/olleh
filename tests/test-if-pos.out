Printing from true!
there is no else
true || false is true!
true && true is true!
true || true is true!
not false is true!
